### PR TITLE
FIX(#5): Update action to use new GITHUB_OUTPUT

### PR DESCRIPTION
**Pull request by thingthing**

### CODE REVIEW

**Comment by NiklasLehnfeld** [comment]
Sorry for the Late reply. I am currently Not using flutter, therefore i can't Test your changes quickly. However, your changes do look wrong anyway. Please See Other comments.

File: action.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: flutter-version-number
-description: 'Reads the version number of the pubspec.yaml'
+description: "Reads the version number of the pubspec.yaml"
 author: NiklasLehnfeld
 branding:
-  icon: 'archive'
```
Comment (NiklasLehnfeld) [inline action.yaml:5]: Changes Like This are Not necessary. It depends on the configuration of the IDE and on personal preferences. I prefer Single quotes, so please Leave it Like this.

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -10,11 +10,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - id: read-version
-        uses: NiklasLehnfeld/flutter-version-number-action@main
+        uses: blackfoot-makers/flutter-version-number-action@main
```
Comment (NiklasLehnfeld) [inline .github/workflows/main.yaml:13]: With This change the Workflow Which Should test This action is Not using This Action anymore - Revert it

File: action.yaml [hunk 3]
```diff
@@ -19,5 +19,5 @@ runs:
     - id: read-yaml
       run: |
         VERSION=`cat ${{inputs.file-path}} | grep -o 'version:[^:]*' | cut -f2 -d":" | xargs`
-        echo "::set-output name=version::$(echo $VERSION)"
+        echo "version=$VERSION" >> "$GITHUB_OUTPUT"
```
Comment (NiklasLehnfeld) [inline action.yaml:22]: I didn't try it, Bit don't you have to Define the Names of the output in the configuration of the Job, as described here: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs ?

**Review by NiklasLehnfeld — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml (hunk 2 above)
Comment (thingthing) [reply .github/workflows/main.yaml:13]: Hey yes, sorry, i wanted to use it in my project so i had to change it in my repository, i forgot about this pull request sorry, i'll close it.

File: action.yaml (hunk 3 above)
Comment (thingthing) [reply action.yaml:22]: Hm no i didn't have to do it, it worked well this way, but i didn't try it recently either, maybe there was an upgrade, i'll look into it, thanks

**Pull request closed without merging**